src/node_crypto.cc

```diff
@@ -741,6 +743,56 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void SecureContext::UseExtraCaCerts(const char* fname) {
+  extra_root_certs_file = fname;
+}
+
+static void emitWarning(Environment* env, const char* fmt, ...) {
+  char warning[1024];
+  va_list ap;
+
+  va_start(ap, fmt);
+  vsnprintf(warning, sizeof(warning), fmt, ap);
+  va_end(ap);
+
+  Local<Object> process = env->process_object();
+  Local<v8::Value> f = process->Get(
+    FIXED_ONE_BYTE_STRING(env->isolate(), "emitWarning"));
+  Local<Value> args[] = {
+    node::OneByteString(env->isolate(), warning, strlen(warning))
+  };
+
+  f.As<v8::Function>()->Call(process, arraysize(args), args);
```

Comment:
I did find this more detailed discussion on the topic from a while back: https://github.com/nodejs/nan/issues/284.  I think the net is that unless you have a specific reason that you don't want process._tickCallback() to be called, then MakeCallback() is preferred over the direct Call.